### PR TITLE
Update documentation for TP website

### LINKED ISSUE #99: Fix issues in tp website
Refer to our website: https://ay2223s1-cs2103t-t09-4.github.io/tp/

1. AddressBook Level-3 should be replaced with Teacher's Pet
image

2. The titles are repeated twice in User Guide and Developer Guide pages
image
image

### PR DESCRIPTION
**Pull request by cadencjk**
Updates documentation to match functions in the current `master` branch. Will still need to update `Adding a student` section when completed. 

Fixes #99

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -194,6 +193,55 @@ Format: `list`
...
+
+Format: `delete INDEX`
+
+- Deletes the person at the specified `INDEX`
+- The index refers to the index number shown in the Student's Details panel (bottom left)
+- The index must be a positive integer. e.g. `1, 2, 3, ...`
+
+Examples:
+- `list` followed by `delete 2` deletes the 2nd person in the Student's Details panel.
+- `find Betsy` followed by `delete 1` deletes the 1st person in the Stundent's Details panel.
```
Comment (gnahzeus) [inline docs/UserGuide.md:228]: For which lines do we include the full-stop? Maybe you can explain the rationale in the description of the PR.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -194,6 +193,55 @@ Format: `list`
...
+---
+### Finding a student : `find`
+
+Finds students whose names contain any of the given keywords.
+
+Format: `find KEYWORD [MORE_KEYWORDS]`
+
+- The search is case-insensitive. e.g. `alex` will match `Alex`
+- The order of the keywords does not matter. e.g. `Yeoh Alex` will match `Alex Yeoh`
+- Only the name is searched.
```
Comment (gnahzeus) [inline docs/UserGuide.md:205]: Similar question to the comment below.

**Review by gnahzeus — COMMENTED** [review]
LGTM, except a question about the use of punctuation.

**Comment by cadencjk** [comment]
> LGTM, except a question about the use of punctuation.

Okay I've standardised the punctuations for user guide. I realised AB3's user guide had these inconsistency in punctuations 🤨.

In short, is everyone okay with the following convention? @AY2223S1-CS2103T-T09-4/developers 
1.  All sentences (> 3 words) should include a punctuation at the end, regardless of whether they are placed in a bullet form.
2.  The only exceptions to 1 are listing features and purely having an `inline code` in a line.
3.  Examples to be abbreviated as ` e.g. `  (note the preceding succeeding spaces).

Some examples: 
    - Additional notes ✅ 
    - Additional notes. ❌ 
    - `delete 3` ✅
    - `delete 3`. ❌ 
    - `list` followed by `delete 2` deletes the 2nd person in the Student's Details panel. ✅
    - `list` followed by `delete 2` deletes the 2nd person in the Student's Details panel ❌
    - The search is case-insensitive. e.g. `alex` will match `Alex`. ✅
    - The search is case-insensitive. eg: `alex` will match `Alex`. ❌

**Review by gnahzeus — APPROVED** [review]
LGTM!

**Pull request merged**